src/serve/hosting.ts

```diff
@@ -1,22 +1,19 @@
-"use strict";
+import clc = require("cli-color");
+import { FirebaseError } from "../error";
+import * as utils from "../utils";
+import * as detectProjectRoot from "../detectProjectRoot";
+import functionsProxy from "../hosting/functionsProxy";
+import cloudRunProxy from "../hosting/cloudRunProxy";
+const superstatic = require("superstatic").server;
```

Comment:
Could this be `import { server } = require("superstatic")`

Note: I always get confused about imports so I could be totally wrong.

Reply:
sorry.'`import { server } = require("superstatic")` cause build error.

I'll read the superstatic implementation 🤔